src/swarm-engine/Swarm/Game/ScenarioInfo.hs

```diff
@@ -164,70 +166,73 @@ orderFileName = "00-ORDER.txt"
 testingDirectory :: FilePath
 testingDirectory = "Testing"
 
-readOrderFile :: (Has (Lift IO) sig m) => FilePath -> m [String]
-readOrderFile orderFile =
-  filter (not . null) . lines <$> sendIO (readFile orderFile)
+readOrderFile :: FilePath -> IO (Maybe [String])
```

Comment:
Why the change to a concrete `IO` type, instead of `Has (Lift IO)`?

Reply:
It's a simple IO function, so using sendIO is only noise.

Reply:
OK, fair enough.